scripts/simplify.py

```diff
@@ -0,0 +1,119 @@
+import logging
+
+import pandas as pd
+import geopandas as gpd
+import xarray as xr
+import boto3
+import s3fs
+from multiprocessing import Pool
+
+logging.basicConfig()
+
+logger = logging.getLogger("gfts")
+logger.setLevel(logging.DEBUG)
+ENDPOINT = "https://s3.gra.perf.cloud.ovh.net/"
+BUCKET = "gfts-ifremer"
+PREFIX = "tags/bargip/tracks_4/"
+PROFILE = "ovh_gfts"
+REGION = "gra"
+NR_OF_CELLS_PER_TIMESLICE = 200
+POOL = 10
+
+
+boto3.setup_default_session(profile_name=PROFILE)
+
+
+def simplify(tag):
+    logger.debug(f"Opening tag {tag}")
+
+    fsg = s3fs.S3FileSystem(
+        anon=False,
+        profile=PROFILE,
+        client_kwargs={
+            "endpoint_url": ENDPOINT,
+            "region_name": REGION,
+        },
+    )
+    store = s3fs.S3Map(
+        root=f"s3://{BUCKET}/{PREFIX}{tag}/states.zarr",
+        s3=fsg,
+        check=False,
+    )
+    data = xr.open_zarr(store=store)
+
+    vars_to_keep = ["states", "latitude", "longitude", "time"]
+    vars_to_drop = [var for var in data.variables if var not in vars_to_keep]
+    data = data.drop_vars(vars_to_drop)
+
+    # Do one by one to save memory. Most timestamps have a lot of nodata
+    # that can be dropped in the loop. If done in one step this will grow fast.
+    dfs = []
```

Comment:
@annefou or @tinaok could you have a look at this loop and let me know if that approach makes sense?

The goal is to keep the N highest probabilities for each timestep in each PDF. This is what we want to keep for visualization.

I am doing it slice by slice to never load the entire PDF into memory. We throw 95% of all the cells away in each step, so in a loop its memory efficient. But maybe there is a way to do this without the for loop? I know in xarray land this should be avoided as much as possible.

cc @danielfdsilva